ssh/tailssh/incubator.go

```diff
@@ -222,13 +222,23 @@ func beIncubator(args []string) error {
 		}
 	}
 
+	homeDir := "/"
```

Comment:
I would be more comfortable if we passed down the desired `homeDir` from the parent process as an argument.